### DISCUSSION TITLE
backend loglevel adjustable?

### DISCUSSION THREAD

**Discussion by drrossum** [mempool/mempool#569] [Q&A]
The backend generates a lot of DEBUG level output.  Is there a way to adjust this?  (Is this the best place to ask questions like this?)

**Comment by visskiss** [comment]
I played with some config and couldn't get anything to change. I also cannot find any documentation on that config file and it seems like many of the settings have no effect.

**Comment by wiz** [comment]
Keep in mind that when you read the README on the top-level of the @mempool/mempool repo on GitHub you are by default viewing the master branch version. To view the README for the latest release version, you need to change the tag to that version.

Currently we have 2 backend configurations you can use to adjust the log priority:

1) "MEMPOOL": { STDOUT_LOG_MIN_PRIORITY": "info" } will adjust what is displayed on the console

2) "SYSLOG": { "MIN_PRIORITY": "info" } will adjust what is sent to syslog on 127.0.0.1 port 514
